core/templates/pages/collection-player-page/collection-player-page.module.ts

```diff
@@ -54,12 +56,14 @@ import { platformFeatureInitFactory, PlatformFeatureService } from
   declarations: [
     OppiaAngularRootComponent,
     CollectionFooterComponent,
-    CollectionNodeListComponent
+    CollectionNodeListComponent,
+    CollectionNavbarComponent
   ],
   entryComponents: [
     OppiaAngularRootComponent,
     CollectionFooterComponent,
-    CollectionNodeListComponent
+    CollectionNodeListComponent,
+    CollectionNavbarComponent
```

Comment:
Everything after OppiaAngularRootComponent has to lexicographically sorted.
```suggestion
    CollectionNavbarComponent
    CollectionNodeListComponent,
```

Reply:
@hngerebara can you please comment "done" when this is done? Thanks!